activesupport/lib/active_support/evented_file_update_checker.rb

```diff
@@ -100,11 +100,12 @@ def execute_if_updated
       def boot!
         normalize_dirs!
 
-        Listen.to(*@dtw, &method(:changed)).start if @dtw.any?
+        @listener = (Listen.to(*@dtw, &method(:changed)) if @dtw.any?)
```

Comment:
```suggestion
        @listener = Listen.to(*@dtw, &method(:changed)) if @dtw.any?
```

Reply:
That is slightly different since it won't always assign `@listener`.  Instead, I rewrote this line using a ternary operator.